gtfs/spec/en/reference.md

```diff
@@ -380,7 +382,7 @@ For a given ordered pair of arriving trip and departing trip, the transfer with
 | `to_trip_id`  | Foreign ID referencing `trips.trip_id` | Optional | Identifies a trip where a connection between routes ends.<br><br>If `to_trip_id` is defined, the transfer will apply to the departing trip for the given `to_stop_id`.<br><br>If both `to_trip_id` and `to_route_id` are defined, the `trip_id` must belong to the `route_id`, and `to_trip_id` will take precedence. |
-|  `transfer_type` | Enum | **Required** | Indicates the type of connection for the specified (`from_stop_id`, `to_stop_id`) pair. Valid options are:<br><br> `0` or empty - Recommended transfer point between routes.<br>`1` - Timed transfer point between two routes. The departing vehicle is expected to wait for the arriving one and leave sufficient time for a rider to transfer between routes.<br>`2` - Transfer requires a minimum amount of time between arrival and departure to ensure a connection. The time required to transfer is specified by `min_transfer_time`.<br>`3` - Transfers are not possible between routes at the location. |
+|  `transfer_type` | Enum | **Required** | Indicates the type of connection for the specified (`from_stop_id`, `to_stop_id`) pair. Valid options are:<br><br> `0` or empty - Recommended transfer point between routes.<br>`1` - Timed transfer point between two routes. The departing vehicle is expected to wait for the arriving one and leave sufficient time for a rider to transfer between routes.<br>`2` - Transfer requires a minimum amount of time between arrival and departure to ensure a connection. The time required to transfer is specified by `min_transfer_time`.<br>`3` - Transfers are not possible between routes at the location.<br>`4` - Passengers can transfer from one trip to another by staying onboard the same vehicle (an "in-seat transfer"). Using this transfer type REQUIRES that the the two trips linked are operated by the same vehicle. When using this type of transfer, it's REQUIRED that both `from_trip_id` and `to_trip_id` values are set. Additionally, the last stop of `from_trip_id` SHOULD be geographically close from the first stop `to_trip_id` and the last arrival time of `from_trip_id` SHOULD be close to the first departure time of `to_trip_id`.  <br>`5` - In-seat transfers are not allowed between sequential trips. The passenger must alight from the vehicle and re-board. Using this transfer type REQUIRES that the the two trips linked are operated by the same vehicle. When using this type of transfer, it's REQUIRED that both `from_trip_id` and `to_trip_id` values are set. Additionally, the last stop of `from_trip_id` SHOULD be geographically close from the first stop `to_trip_id` and the last arrival time of `from_trip_id` SHOULD be close to the first departure time of `to_trip_id`. |
```

Comment:
> for the specified (`from_stop_id`, `to_stop_id`, `from_trip_id`, `to_trip_id`, `from_route_id`, `to_route_id`) pair.

Reply:
Good point @timMillet. Updated. Didn't touch stop_ids because trip transfers are also valid with the other types. (Anyway probably out of scope for this PR?)